spec/rspec/mocks/any_instance_spec.rb

```diff
@@ -1213,6 +1230,17 @@ def dup(_)
         end
       end
 
+      context "on a grandchild class" do
+        it "stubs a method not implemented in grandchild" do
+          allow_any_instance_of(::AnyInstanceSpec::ChildClass).to receive(:foo).and_return :result
+          expect(::AnyInstanceSpec::ChildClass.new.foo).to eq :result
+        end
+
+        it "stubbed method doesn't get restored to grandchild" do
```

Comment:
It's nice to phrase the test descriptions to read gramatically well off of "it", and "it stubbed method..." does not read well.  How about `it "does not restore stubbed method to grandchild"`?